docs/quickstart.ipynb

```diff
@@ -1393,11 +1879,42 @@
     }
    ],
    "source": [
-    "LetsPlot.setup_html()\n",
-    "\n",
     "multi_fit.coefplot().show()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can use a `matplotlib` backend as well (**tip** make sure to specify the `figsize` as the `LetsPlot` defauls are very large):"
```

Comment:
I noticed this the other day as well but then forgot! =) Would it be possible to change the figsize defaults so that we produce sensibly scaled figures for both searborn and let's plot out of the box? Should of course be a separate issue then.

Reply:
Indeed! Created an issue! https://github.com/s3alfisc/pyfixest/issues/441

Reply:
Seen it =) Thank you!